Amazon.QLDB.Driver/transaction/BaseTransaction.cs

```diff
@@ -66,5 +67,16 @@ internal static QldbHash Dot(QldbHash seed, string statement, List<IIonValue> pa
 
             return seed.Dot(statementHash);
         }
+
+        internal static QldbHash Dot(QldbHash seed, string statement, ValueHolder[] parameters)
```

Comment:
Why is this here/why do we need this?

Reply:
It is to support the new [Transaction.execute()](https://github.com/awslabs/amazon-qldb-driver-dotnet/blob/object-serialization-idda-jun-4-2021/Amazon.QLDB.Driver/transaction/AsyncTransaction.cs#L148-L153) functions that take in the query object with `ValueHolder[]` as the parameters.